### PR TITLE
[DEVOPS-407]  update-proposal | cdn:  use CDN urls for the website

### PR DESCRIPTION
**Pull request by deepfire**

### CODE REVIEW

File: iohk/common/Utils.hs [hunk 1]
```diff
@@ -112,3 +114,8 @@ jsonLowerStrip n = AE.genericToJSON $ AE.defaultOptions { AE.fieldLabelModifier
 s3Link :: Region -> BucketName -> ObjectKey -> Text
 s3Link region (BucketName bucket) (ObjectKey key) =
   mconcat [ "https://s3-", AWS.toText region, ".amazonaws.com/" , bucket, "/", key ]
+
+cdnLink :: HasCallStack => Environment -> ObjectKey -> Text
```
Comment (cleverca22) [inline iohk/common/Utils.hs]: what about making this an entry in the `config.yaml` so development clusters support updating
this code makes it harder to test the whole update system in a disposable cluster

Comment (deepfire) [reply iohk/common/Utils.hs]: To be fair, this code doesn't really change the status quo -- we had `s3Link` to the same effect previously.

Comment (deepfire) [reply iohk/common/Utils.hs]: I agree that your proposal is desirable, though!

Comment (deepfire) [reply iohk/common/Utils.hs]: Done..

**Pull request merged**